app/src/main/res/layout/sheet_phone_mask.xml

```diff
@@ -15,7 +15,6 @@
     android:layout_marginEnd="@dimen/spacing_16"
     android:layout_marginStart="@dimen/spacing_16"
     android:gravity="center"
-    android:maxLines="1"
```

Comment:
![Image](https://user-images.githubusercontent.com/1978065/58793561-00a82c00-8614-11e9-9805-7c37b13b3c95.jpg)

The text looks weird when it wraps to multiple lines. What do you think about setting the gravity to `start` instead?

Reply:
I'm not sure. It'd look even weird. Can you please try it out?

Reply:
Tried. `center` is better. Let's keep as is.